uefi/src/table/boot.rs

```diff
@@ -202,6 +201,7 @@ pub struct BootServices {
 
     // Misc services
     get_next_monotonic_count: usize,
+    install_configuration_table: extern "efiapi" fn(entry_guid: Guid, table_ptr: *const c_void) -> Status,
```

Comment:
Was moving this function in the table intentional? Unless I'm misreading the spec, the original location above is correct.

Reply:
Fixed